recipes/openjph/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+context:
+  version: "0.17.0"
+
+package:
+  name: openjph
+  version: ${{ version }}
+
+source:
+  url: https://github.com/aous72/OpenJPH/archive/${{ version }}.tar.gz
+  sha256: 9cd09a5f3a8046b10bded787212afd2410836f9c266964a36f61dc4b63f99b6c
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - ${{ compiler('cxx') }}
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    - cmake
+    - if: win
+      then:
+        - ninja
+      else:
+        - make
+  host:
+    - libtiff
+  run_exports:
+    - ${{ pin_subpackage('openjph', upper_bound='x.x') }}
+
+tests:
+  - package_contents:
+      include:
+        - openjph/ojph_version.h
+  - script:
+    - if: unix
+      then:
+        - test -f ${PREFIX}/include/openjph/ojph_version.h
+        - test -f ${PREFIX}/lib/libopenjph${SHLIB_EXT}
```

Comment:
You could add these tests to package contents as well (although no harm to test them twice for the time being):

```
bin:
  - ojph_compress
  - ojph_expand
lib:
  - openjph
```

Reply:
thanks.